modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2337,8 +2337,17 @@ export class Wallet implements IWallet {
     if (this._wallet.multisigType === 'tss') {
       return await this.sendManyTss(params);
     } else {
-      const signedPrebuild = await this.prebuildAndSignTransaction(params);
-      return await this.submitTransaction(signedPrebuild);
+      switch (this._wallet.type) {
+        case 'hot':
+          const signedPrebuild = await this.prebuildAndSignTransaction(params);
+          return await this.submitTransaction(signedPrebuild);
+        case 'cold':
+          // Don't sign or send and just return the built transaction
+          return params.txPrebuild;
+        case 'custodial':
+          const url = this.baseCoin.url('/wallet/' + this.id() + '/tx/initiate');
+          return await this.bitgo.post(url).send(params.prebuildTx.buildParams).result();
+      }
```

Comment:
Can we update the jsdocs to let users know of the different send/return scenarios in party because of the `:Promise<any>` return.

Reply:
Updated